python_requires should be specified

<!--
Welcome to the Jaeger project! 👋🎉

- Please search for existing issues to avoid creating duplicate bugs/feature requests.
- Please be respectful and considerate of others when commenting on issues.
- Please provide as much information as possible so we all understand the issue.
- If you only have a question, you may get a faster response by asking in
    - our chat room https://gitter.im/jaegertracing/Lobby, or
    - the forum https://groups.google.com/d/forum/jaeger-tracing
    (but please don't double post)
-->

## Requirement - what kind of business use case are you trying to solve?

Installing jaeger-client in Python 2

## Problem - what in Jaeger blocks you from solving the requirement?

jaeger-client>=4.5.0 is not support Python 2.x, but still could installed by pip on this Python because lack of `python_requires` in setup.py.

It cause the following problem:
```
    from .codecs import TextCodec, ZipkinCodec, ZipkinSpanFormat, BinaryCodec
.../.tox/py27/lib/python2.7/site-packages/jaeger_client/codecs.py:32: in <module>
    import urllib.parse
E   ImportError: No module named parse
```


## Proposal - what do you suggest to solve the problem or improve the existing situation?

`python_requires` should be specified in setup.py to prevent pip from installing new packages for Python 2.x

## Any open questions to address

--- a/setup.py
+++ b/setup.py
@@ -35,6 +35,7 @@ setup(
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ],
+    python_requires='>=3.5',
     install_requires=[
         'threadloop>=1,<2',
         'thrift',

